CHANGES.md

```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+## 1.11.0 (core) / 0.27.0 (libraries)
+
+## Major changes since 1.10.0 (core) / 0.26.0 (libraries)
+
+### Components — Configurable, reusable building blocks for data pipelines (now stable)
+
+Components, first released as Preview in 1.10.6, have reached Release Candidate status. APIs are stable, fully documented, and are the recommended, product-ready defaults for new Dagster projects.
+
+- **Declarative spec:** A short YAML block in `defs.yaml`, or a lightweight `Component` subclass in Python, lets you spin up assets, resources, schedules, IO managers, and checks, removing boilerplate while keeping every definition type-checked.
+- **Custom Components:** Subclassing `Component` lets you wrap any internal script or SaaS behind a strongly-typed interface; get the same autocompletion and docs as first-party integrations.
```

Comment:
not sure if I would describe it as "any internal script or SaaS", maybe "any internal script or third-party tool"?